Cargo.toml

```diff
@@ -1,11 +1,27 @@
 [package]
 name = "blazesym"
-description = "BlazeSym is a library that symbolizes addresses where symbol names, source file names, and line numbers can be acquired."
-version = "0.1.0"
-authors = ["Kui-Feng <thinker.li@gmail.com>"]
+description = "blazesym is a library that can be used for address symbolization and more."
+version = "0.2.0-alpha.0"
+edition = "2021"
+rust-version = "1.63"
+authors = ["Daniel Müller <deso@posteo.net>", "Kui-Feng <thinker.li@gmail.com>"]
 license-file = "LICENSE"
 repository = "https://github.com/libbpf/blazesym"
-edition = "2021"
+readme = "README.md"
+categories = [
+  "algorithms",
+  "api-bindings",
+  "development-tools::debugging",
+  "os::unix-apis",
+  "value-formatting",
+]
+keywords = [
+  "dwarf",
+  "elf",
+  "gsym",
+  "symbolization",
+  "tracing",
```

Comment:
stacktrace as well?

Reply:
We only have a [limited number](https://doc.rust-lang.org/cargo/reference/manifest.html?highlight=keywords#the-keywords-field) of keywords we can pick. Want me to replace `tracing`?

Reply:
argh, hard to pick, but I think `stacktrace` is super relevant, so would keep it. Is "symbolization" a popular one? I'd imagine most people think "stack trace" instead of "symbolization", so maybe that one? "tracing" seems relevant, so I'd keep it?

Reply:
> Is "symbolization" a popular one?

[We are the first](https://crates.io/keywords/symbolization)! :P

> I'd imagine most people think "stack trace" instead of "symbolization", so maybe that one? "tracing" seems relevant, so I'd keep it?

Okay. Will replace `symbolization`.  `tracing` is currently mostly used for the crate of the [same name](https://crates.io/crates/tracing).